iceoryx_posh/include/iceoryx_posh/internal/roudi/port_manager.hpp

```diff
@@ -62,7 +62,6 @@ class PortManager
 
     virtual ~PortManager() noexcept = default;
 
-    /// @todo Remove this later
```

Comment:
@elfenpiff Shouldn't this be taken to an issue where the introspection can be disabled with a compile time switch?

Reply:
Do you have an issue number for me. Otherwise I would remove it, this has to be refactored heavily and we also had enough issues in the code which were pointing to already closed issues.

Reply:
@elfenpiff I suppose, #518 would fit best here. An alternative would also be #414